apps/deploy-web/src/context/CustomChainProvider/CustomChainProvider.tsx

```diff
@@ -13,6 +13,16 @@ import { akash, akashSandbox, akashTestnet, assetLists } from "@src/chains";
 import { useSelectedNetwork } from "@src/hooks/useSelectedNetwork";
 import { customRegistry } from "@src/utils/customRegistry";
 
+import "@interchain-ui/react/styles";
+import "@interchain-ui/react/globalStyles";
```

Comment:
This is already imported at the top of the file

Reply:
Fixed! @baktun14